backend-api/src/functions/asyncActiveSession/tokenService/tests/publicKeyGetter.test.ts

```diff
@@ -0,0 +1,107 @@
+import { IJwks, IPublicKeyGetter, PublicKeyGetter } from "../publicKeyGetter";
+import { MockJWTBuilder } from "../../../testUtils/mockJwtBuilder";
+
+describe("Public Key Getter", () => {
+  let mockJwks: IJwks;
+  let publicKeyGetter: IPublicKeyGetter;
+  let mockEncodedJwt: string;
+
+  beforeEach(async () => {
+    mockEncodedJwt = new MockJWTBuilder()
+      .setKid("mockKid")
```

Comment:
This builder is shared between lambdas - the `kid` in the header of the JWT is not use in `asyncCredential` token service tests. I wanted to add a `mockKid` as a default into the builder so that it doesn't need to be set like this but this breaks the `asyncCredential` token service test because it changes the signature. Is it worth updating the `MockJWTBuilder` so that it's easier to update the signature when needed?